### ISSUE TITLE
v3.6.2 - Image cropping no longer works with AWS S3 bucket

### ISSUE THREAD

**Issue by AlexViapiano** [strapi/strapi#10333]
### Describe the bug

I'm using the latest Strapi version 3.6.2 and ever since I starting using AWS S3 bucket as my image upload provider,  the image cropping doesn't work. Once you click the button, I see the X and checkmark, however you can't click and drag the area to be cropped. The whole picture has no fade, but usually with cropping there a dark filter over the area to be cropped. The download button also doesn't work but the delete and link button works as expected. Here is my plugins.js:

`module.exports = ({ env }) => ({
email: {
provider: "sendgrid",
providerOptions: {
apiKey: env("SENDGRID_API_KEY"),
},
settings: {
defaultFrom: "support@waytoogood.com",
defaultReplyTo: "support@waytoogood.com",
},
},
upload: {
provider: "aws-s3",
providerOptions: {
accessKeyId: env("AWS_ACCESS_KEY_ID"),
secretAccessKey: env("AWS_ACCESS_SECRET"),
region: env("AWS_REGION"),
params: {
Bucket: env("AWS_BUCKET"),
},
},
},
sentry: {
dsn: env("SENTRY_DSN"),
},
});`

### Steps to reproduce the behavior

1. Go to content-type w/ media & add a picture
2. Once added, hover picture and click on crop
3. Then click crop button and see the problem

### Expected behavior

I expect to be able to crop and image and have is edit/save properly.

### Screenshots

![Screen Shot 2021-05-16 at 10 28 53 PM](https://user-images.githubusercontent.com/28673592/118525803-978d2680-b70d-11eb-884d-29652727b2bb.png)

### System

- Node.js version: v12.20.0
- NPM version: 6.14.8
- Strapi version: 3.6.2
- Database: mySQL
- Operating system: macOS

**Comment by derrickmehaffy** [comment]
I can confirm this on v3.6.2: https://www.loom.com/share/8e086def2af442f499e154cdf7f68d4e

**Comment by derrickmehaffy** [comment]
Just tested again today and could not reproduce it on Mac, Linux, or Windows :thinking: really strange

**Comment by WTGdev** [comment]
Thats strange considering you reproduced it 7 days ago in that loom video!

I’m can reproduce in both mac and windows, using chrome browser.

**Comment by derrickmehaffy** [comment]
Agreed it's quite odd @WTGdev but for some reason I cannot now, maybe an external depends got locked?

Try deleting your package-lock.json and yarn.lock along with node_modules and reinstall the modules and rebuild the admin? (Might be good to copy your lock file over and compare the two?)

**Comment by WTGdev** [comment]
I just tried deleting package-lock.json, node_modules & rebuilt admin but the issue still occurs.

When you tried testing today were you still using aws s3 as your provider?

  upload: {
    provider: "aws-s3",
    providerOptions: {
      accessKeyId: env("AWS_ACCESS_KEY_ID"),
      secretAccessKey: env("AWS_ACCESS_SECRET"),
      region: env("AWS_REGION"),
      params: {
        Bucket: env("AWS_BUCKET"),
      },
    },
  },

**Comment by WTGdev** [comment]
I got a solution!

I had a buddy of mine check my configuration for s3 bucket and he edited the s3 cors policy on the bucket. So basically you need to specify your url in the AllowedOrigins for it to work!

[
    {
        "AllowedHeaders": [
            "*"
        ],
        "AllowedMethods": [
            "GET",
            "HEAD"
        ],
        "AllowedOrigins": [
            "https://_________________.com",
            "https://_________________.com"
        ],
        "ExposeHeaders": [
            "Authorization"
        ],
        "MaxAgeSeconds": 3000
    }
]

**Comment by derrickmehaffy** [comment]
Ah @WTGdev I think your right, I loaded up an S3 bucket with extremely relaxed CORS controls (basically allow anything from anywhere) which is why I couldn't reproduce it.

Going to move this over to the documentation repo so we can document the S3 provider settings to note you need to configure CORS on the bucket.
